skyVegasTests/depositFLowGameClient.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, type Page } from "@playwright/test";
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+import { config } from "../config/config";
+import { startEventListener } from "./utilities/starteventListenerHelper";
+
+test.describe.configure({ mode: "serial" });
+const consoleMessages: string[] = [];
+let expectedMessages;
+let page: Page;
+const games = JSON.parse(fs.readFileSync("games.json", "utf-8"));
+games.forEach((game) => {
+  test.describe(`Testing with text: ${game}`, () => {
+    test.beforeAll(async ({ browser }) => {
+      const data = fs.readFileSync("ExpectedSlotConsoleMessages.json", "utf-8");
+      expectedMessages = JSON.parse(data);
+      page = await browser.newPage();
+      await login(page);
+    });
+
+    test.afterAll(async () => {
+      await page.close();
+    });
+
+    test("Test deposit flow", async ({browser}) => {
+        startEventListener(page, consoleMessages);
+        await launchGame(page, game, consoleMessages);
+        await page
+        .frameLocator('#root iframe')
+        .getByRole('button', { name: 'Deposit' })
+        .click();
+        const userCreationPage = await browser.newPage();
```

Comment:
User creation could go onto a helper class? There is a chance we can use it for other brands like pp etc

Reply:
Done